pytorch_blade/torch_blade_build.py

```diff
@@ -62,6 +62,10 @@ def __init__(
         self.blade_gemm = check_env_flag(
             "TORCH_BLADE_BUILD_BLADE_GEMM", "OFF"
         )
+        self.skip_disc_cmd_build = check_env_flag(
+            "TORCH_BLADE_SKIP_DISC_CMD_BUILD", "OFF"
```

Comment:
where to use this env var? I can not find usage in this PR.

Reply:
Will be used when building. It can be set global.

Reply:
I can not find `TORCH_BLADE_SKIP_DISC_CMD_BUILD` from the codebase except here, for what I mean, we should bring in the flags as little as possible, If we can not enable this flag, always making this flag `OFF` is better.